src/mmw/js/src/analyze/constants.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var constants = {
+    csvColumnMaps: {
+        nlcd: {
+            'nlcd': 'nlcd_id',
+            'area': 'area_sq_m',
+            'coverage': 'coverage_pct',
+        },
+        soil: {
+            'code': 'soil_code',
+            'area': 'area_sq_m',
+            'coverage': 'coverage_pct'
+        },
+        animals: {
+            'aeu': 'animal_count',
+            'type': 'animal_type'
+        },
+        pointSource: {
+            'kgp_yr': 'phosphorus_load_kg_per_yr',
+            'mgd': 'discharge_million_gallon_per_day',
+            'kgn_yr': 'nitrogen_load_kg_per_yr'
+        },
+        waterQuality: {
+            'nord': 'nord',
+            'areaha': 'area_hectares',
+            'geom': 'geometry',
```

Comment:
Good catch; I removed it from the data before it gets run through these keys, but I'll cut it here too.